src/App.js

```diff
@@ -1,24 +1,31 @@
-import logo from './logo.svg';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import Books from './components/books';
+import Categories from './components/categories';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div id="navlinks">
+        <div className="nav-link">
+          <Link to="/">Books</Link>
+        </div>
+        <div className="nav-link">
+          <Link to="/categories">Categories</Link>
+        </div>
+      </div>
+
+      <Switch>
+        <Route exact path="/">
+          <Books />
+        </Route>
+        <Route path="/categories">
+          <Categories />
+        </Route>
+      </Switch>
+    </Router>
   );
```

Comment:
- [ ] Let us restructure our implementation a little bit, I would recommend creating a `navbar file` inside the component to keep our link and app.js to keep our routes.